packages/element/src/test/create-interpolate-element.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Internal dependencies
+ */
+import { createElement, Fragment } from '../react';
+import createInterpolateElement from '../create-interpolate-element';
+
+describe( 'createInterpolateElement', () => {
+	it( 'returns same string when there is no conversion map', () => {
+		const testString = 'This is a string';
+		expect(
+			createInterpolateElement( testString, [] )
+		).toEqual( 'This is a string' );
+	} );
+	it( 'returns same string when there are no tokens in the string', () => {
+		const testString = 'This is a string';
+		expect(
+			createInterpolateElement(
+				testString,
+				{ someValue: { value: 10 } }
+			)
+		).toEqual( <Fragment>{ testString }</Fragment> );
```

Comment:
It seems like we should be testing here to that we `toEqual( testString )` and not have the `<Fragment />`